src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -1757,12 +1757,12 @@ int MPL_gpu_ipc_handle_unmap(void *ptr)
     ret = zeMemGetAllocProperties(ze_context, ptr, &ptr_attr, &device);
     ZE_ERR_CHECK(ret);
 
-    dev_id = device_to_dev_id(device);
-    if (dev_id == -1) {
-        goto fn_fail;
-    }
-
     if (likely(MPL_gpu_info.specialized_cache)) {
+        dev_id = device_to_dev_id(device);
+        if (dev_id == -1) {
+            goto fn_fail;
+        }
```

Comment:
For this commit - "mpl/gpu/ze: fix issue when disabling specialized cache", please in commit message explain what was the issue

Reply:
I don't recall why I worded the commit message this way.. I don't see a reason this change is needed specifically when disabling the specialized cache. I will reword the message to better reflect the change